spec/mobility/plugins/active_record/dirty_spec.rb

```diff
@@ -233,22 +239,31 @@ def clear_changes_information
         expect(article.title_was).to eq("foo")
 
         article.save
-        if ENV['RAILS_VERSION'].present? && ENV['RAILS_VERSION'] < '5.0'
-          expect(article.title_changed?).to eq(nil)
-        else
+
+        expect(article.title_changed?).to eq(false)
+
+        if ENV['RAILS_VERSION'].present? && ENV['RAILS_VERSION'] >= '5.0'
           expect(article.title_previously_changed?).to eq(true)
           expect(article.title_previous_change).to eq(["foo", "bar"])
           expect(article.title_changed?).to eq(false)
-        end
 
-        # AR-specific suffix methods
-        if ENV['RAILS_VERSION'].present? && ENV['RAILS_VERSION'] > '5.0'
-          expect(article.saved_change_to_title?).to eq(true)
-          expect(article.saved_change_to_title).to eq(["foo", "bar"])
-          expect(article.title_before_last_save).to eq("foo")
-          expect(article.will_save_change_to_title?).to eq(false)
-          expect(article.title_change_to_be_saved).to eq(nil)
-          expect(article.title_in_database).to eq("bar")
+          # AR-specific suffix methods, added in 5.1
+          if ENV['RAILS_VERSION'] > '5.0'
+            expect(article.saved_change_to_title?).to eq(true)
+            expect(article.saved_change_to_title).to eq(["foo", "bar"])
+            expect(article.title_before_last_save).to eq("foo")
+            #expect(article.will_save_change_to_title?).to eq(false)
+            expect(article.title_change_to_be_saved).to eq(nil)
+            expect(article.title_in_database).to eq("bar")
+
+            # attribute handlers
+            expect(article.saved_change_to_attribute?(:title_en)).to eq(true)
+            #expect(article.saved_change_to_attribute(:title_en)).to eq(['foo', 'bar'])
+            #expect(article.attribute_before_last_save(:title_en)).to eq('foo')
+            #expect(article.will_save_change_to_attribute?(:title_en)).to eq(false)
+            #expect(article.attribute_change_to_be_saved(:title_en)).to eq(nil)
+            #expect(article.attribute_in_database(:title_en)).to eq('bar')
```

Comment:
@pwim These specs all currently fail on 0-8-stable, so these other attribute methods (`saved_change_to_attribute`, etc.) don't work with translated attributes, just `saved_change_to_attribute?`. I'm not exactly sure why but the fact that they worked or not was pretty much dependent on internals of AR/AM.

These other specs will now *pass* with the new changes (once I commit them here), and they will be independent of those AM/AR internals.